blocks/query/edit.tsx

```diff
@@ -319,9 +341,49 @@ export default function Edit({
                   onSelect={(newCategories: Term[]) => setTerms(taxonomy, newCategories)}
                   multiple
                 />
+                {terms[taxonomy]?.length > 1 ? (
+                  <SelectControl
+                    label={sprintf(
+                      __('%s Relation', 'wp-curate'),
+                      availableTaxonomies[taxonomy].name || taxonomy,
+                    )}
+                    help={__('AND: Posts must have all selected terms. OR: Posts may have one or more selected terms.', 'wp-curate')}
+                    options={[
+                      {
+                        label: __('AND', 'wp-curate'),
```

Comment:
Curious, if I needed to change this label: would that be possible?

Reply:
@renatonascalves I can probably move it to PHP and make it filterable, then expose it to JS with `wp_localize_script()`.
Do you think you will need to?